src/crimson/osd/shard_services.cc

```diff
@@ -188,21 +188,21 @@ seastar::future<> ShardServices::send_pg_temp()
   if (pg_temp_wanted.empty())
     return seastar::now();
   logger().debug("{}: {}", __func__, pg_temp_wanted);
-  boost::intrusive_ptr<MOSDPGTemp> ms[2] = {nullptr, nullptr};
+  MURef<MOSDPGTemp> ms[2] = {MURef<MOSDPGTemp>{nullptr, TOPNSPC::common::unique_ptr_release}, MURef<MOSDPGTemp>{nullptr, TOPNSPC::common::unique_ptr_release}};
```

Comment:
This is too long in a single line, and probably worth a method like `MURef<T> crimson::net::make_null_message<T>()`.